lib/msal-node/src/cache/serializer/Deserializer.ts

```diff
@@ -45,14 +54,19 @@ export class Deserializer {
      * Deserializes id tokens to IdTokenEntity objects
      * @param idTokens
      */
-    static deserializeIdTokens(idTokens: StringDict | undefined): IdTokenCache {
+    static deserializeIdTokens(idTokens: Record<string, SerializedIdTokenEntity>): IdTokenCache {
         const idObjects: IdTokenCache = {};
         if (idTokens) {
-            Object.keys(idTokens).map(function (key) {
-                const mappedIdT = CacheHelper.renameKeys(
-                    idTokens[key],
-                    IdTokenCacheMaps.fromCacheMap
-                );
+            Object.keys(idTokens).map(function(key) {
+                const serializedIdT = idTokens[key];
+                const mappedIdT = {
+                    homeAccountId: serializedIdT.home_account_id,
+                    environment: serializedIdT.environment,
+                    credentialType: serializedIdT.credential_type,
+                    clientId: serializedIdT.client_id,
+                    secret: serializedIdT.secret,
+                    realm: serializedIdT.realm,
+                };
                 const idToken: IdTokenEntity = new IdTokenEntity();
                 CacheHelper.toObject(idToken, mappedIdT);
                 idObjects[key] = idToken;
```

Comment:
This can be addressed later, but I would use reduce here:

```js
return Object.keys(idTokens).reduce((idObjects, key) => {
  // ...
  idObjects[key] = idToken;
  return idObjects;
}, {});

Reply:
I remember trying `reduce` and it not working. Can check again and as you said will do it  in a different PR.